src/dotnet/scripts/vendor/dotnet-install.sh

```diff
@@ -1447,14 +1447,14 @@ generate_regular_links() {
 
     if [ "$valid_legacy_download_link" = true ]; then
         say_verbose "Constructed legacy named payload URL: $legacy_download_link"
-    
+
         download_links+=($legacy_download_link)
         specific_versions+=($specific_version)
         effective_versions+=($effective_version)
         link_types+=("legacy")
     else
         legacy_download_link=""
-        say_verbose "Cound not construct a legacy_download_link; omitting..."
+        say_verbose "Could not construct a legacy_download_link; omitting..."
```

Comment:
Can we revert this due to https://github.com/devcontainers/features/blob/89280f408f04aa113760e7e61fba4522a28ea454/src/dotnet/scripts/vendor/README.md#important-note as it will get overwritten soon! 

This is definitely an err, would be nice to be corrected in upstream.

Reply:
Reverted